tests/acceptance/expected-failures-with-oc10-server copy-openid-login.md

```diff
@@ -22,7 +22,7 @@ Other free text and markdown formatting can be used elsewhere in the document if
 -   [webUIPreview/imageMediaViewer.feature:154](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUIPreview/imageMediaViewer.feature#L154)
 
 ### [WebUI Login](https://github.com/owncloud/web/issues/4677)
```

Comment:
is that filename intended? `tests/acceptance/expected-failures-with-oc10-server copy-openid-login.md`

Reply:
Mistake while copy pasting, i'll fix that